ftl/common/cache.py

```diff
@@ -99,8 +104,25 @@ def Get(self, base_image, namespace, checksum):
             logging.info('No cached base image found for entry: %s.' % entry)
         return None
 
-    def Store(self, base_image, namespace, checksum, value):
-        entry = self._tag(base_image, namespace, checksum)
+    def GetAndValidate(self, base, namespace, cache_key):
```

Comment:
Maybe "GetAndCheckTTL"? Validate is pretty generic.

Reply:
done